src/scripts/build_tts_dataset.py

```diff
@@ -23,10 +24,12 @@ def main(config: DictConfig) -> None:
     """
     # Get the individual datasets
     raw_dir = Path(config.dirs.data) / config.dirs.raw
-    datasets = {
-        name: builder(output_dir=raw_dir)
-        for name, builder in ALL_DATASET_BUILDERS.items()
-    }
+    datasets: dict[str, list[str]] = defaultdict()
+    for name, builder in ALL_DATASET_BUILDERS.items():
+        try:
+            datasets[name] = builder(output_dir=raw_dir)
+        except TypeError:
+            datasets[name] = builder(cfg=config, output_dir=raw_dir)
```

Comment:
Maybe all builders should just take the config as argument, to make them more flexible, and make this handling more neat - I'll make that as a separate PR after this is merged in.